Classes/Service/ParserService.php

```diff
@@ -362,7 +362,7 @@ protected function regexParser(&$text, Term $term, &$replacements, $wrappingCall
          */
         $regex = self::REGEX_DELIMITER .
             '(^|\G|[\s\>[:punct:]]|\<br*\>)' .
-            '(' . preg_quote($term->getName(), self::REGEX_DELIMITER) . ')' .
+            preg_quote($this->settings['termPrefix']) . '(' . preg_quote($term->getName(), self::REGEX_DELIMITER) . ')' .
```

Comment:
Missing the delimiter as second parameter
see e87eb34a82440405dc5f80fe44ec1e324717b187 why